lib/opensusebasetest.pm

```diff
@@ -49,7 +49,7 @@ sub clear_and_verify_console {
     my ($self) = @_;
 
     clear_console;
-    assert_screen('cleared-console');
+    assert_screen('cleared-console') unless testapi::is_serial_terminal();
```

Comment:
please use: 
`use testapi qw(is_serial_terminal :DEFAULT);`

so that you can call directly `is_serial_terminal()`